sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md

```diff
@@ -262,17 +256,37 @@ To build and install the starter use the following command line:
   mvn clean install -DskipTests=true
 ```
 
+## Troubleshooting
+### Enable client logging
+Azure SDKs for Java offers a consistent logging story to help aid in troubleshooting application errors and expedite their resolution. The logs produced will capture the flow of an application before reaching the terminal state to help locate the root issue. View the [logging][logging] wiki for guidance about enabling logging.
+
+### Enable Spring logging
+Spring allow all the supported logging systems to set logger levels set in the Spring Environment (for example, in application.properties) by using `logging.level.<logger-name>=<level>` where level is one of TRACE, DEBUG, INFO, WARN, ERROR, FATAL, or OFF. The root logger can be configured by using logging.level.root.
 
-# Azure Key Vault Certificates client library for Java
+The following example shows potential logging settings in `application.properties`:
 
-# Getting started
+```properties
+logging.level.root=WARN
+logging.level.org.springframework.web=DEBUG
+logging.level.org.hibernate=ERROR
+```
 
-# Key concepts
+For more information about setting logging in spring, please refer to the [official doc](https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-logging).
 
-# Examples
+## Next steps
+The following section provide a sample project illustrating how to use the starter.
+### More sample code
+- [Key Vault Certificates][sample]
```

Comment:
Move `### More sample code` to `## Examples`.

Reply:
The subsection here that displays a sample link follows sdk readme specifications, please refer to all other Spring READMEs.